luigi/batch_notifier.py

```diff
@@ -0,0 +1,201 @@
+import collections
```

Comment:
That is the case, although it probably wouldn't be too hard to make it a standalone library if someone wanted to use it like that.